docs/new-project-setup.md

```diff
@@ -0,0 +1,305 @@
+# New Project Setup
+
+The DSE team regularly creates new Snowflake accounts in our Snowflake org.
+We do this instead of putting all of our data projects into our main account for a few reasons:
+
+1. At the end of a project, we often want to transfer account ownership to our partners.
+    Having it separated from the start helps that process.
+1. We frequently want to add our project champion or IT partners to our account as admins.
+    This is safer if project accounts are separate.
+1. We often want to have accounts in a specific cloud and region for compliance or data transfer regions.
+1. Different projects may require different approaches to account-level operations like OAuth/SAML.
+
+Here we document the steps to creating a new Snowflake account from scratch.
+
+## Prerequisites
+
+### Obtain permissions in Snowflake
+
+In order to create a new account, you will need access to the `orgadmin` role.
+If you have `accountadmin` in the primary Snowflake account, you can grant it to yourself:
+
+```sql
+USE ROLE accountadmin;
+GRANT ROLE orgadmin TO USER <YOUR-USER>;
+```
+
+If you later want to revoke the `orgadmin` role from your user or any other, you can do so with:
+
+```sql
+USE ROLE accountadmin;
+REVOKE ROLE orgadmin FROM USER <YOUR-USER>;
+```
+
+### Get access to AWS
+
+We typically create our Snowflake architecture using Terraform.
+Terraform state is stored in S3 buckets within our AWS account,
+so you will need read/write access to those buckets.
+
+Ask a DSE AWS admin to give you access to these buckets,
+and [configure your AWS credentials](./setup.md#aws).
+
+
+### Install terraform dependencies
+
+You can install Terraform using whatever approach makes sense for your system,
+including using `brew` or `conda`.
+
+Here is a sample for installing the dependencies using `conda`:
+
+```bash
+conda create -n infra python=3.10  # create an environment named 'infra'
+conda activate infra  # activate the environment
+conda install -c conda-forge terraform tflint  # Install terraform and tflint
+```
+
+## Snowflake account setup
+
+### Create the account
+
+1. Assume the `ORGADMIN` role
+1. Under the "Admin" side panel, go to "Accounts" and click the "+ Account" button:
+    1. Select the cloud and region appropriate to the project. The region should be in the United States.
+    1. Select "Business Critical" for the Snowflake Edition.
+    1. You will be prompted to create an initial user with `ACCOUNTADMIN` privileges. This should be you.
+        You will be prompted to create a password for your user. Create one using your password manager,
+        but know that it will ask you change your password upon first log-in.
+    1. Save the Account Locator and URL for your new account.
```

Comment:
```suggestion
    1. Save the Account Locator and Account Locator URL for your new account.
```
Account URL and Account Locator URL are different so I am suggesting this additional language for more clarity